render.yaml

```
@@ -0,0 +1,245 @@
+services:
+  ## Explorer frontend
+  - type: web
+    name: frontend
+    env: node
+    repo: https://github.com/near/near-explorer.git
+    pullRequestPreviewsEnabled: true
+    buildCommand: cd frontend && npm ci && npm run build
```

I was confused by `npm ci`, so I suggest we use full names of the commands:

```suggestion
    buildCommand: cd frontend && npm clean-install && npm run build
```

---

`npm ci` - 8M [mentions](https://github.com/search?q=npm+ci&type=code) in github code
`npm clean-install` - 2K [mentions](https://github.com/search?q=npm+clean-install&type=code) in github code

It may be not a very precise result (as `npm` and `ci` could be located nearby by coincidence), but there is more than 4 orders of magnitude difference.

I believe `npm ci` is widely spread :)

---

> `npm ci` - 8M [mentions](https://github.com/search?q=npm+ci&type=code) in github code

Have you checked the search results? There is no `"npm ci"` on the first page, and all the usages are actually about `npm run ci:test` or something along those lines, which is exactly why I suggested to be explicit about `npm clean-install` since I was confused how npm could do anything related to continuous integration as its base command.

> npm clean-install - 2K [mentions](https://github.com/search?q=npm+clean-install&type=code) in github code

These are all legit CI pipelines where clean-install is used explicitly

---

> Have you checked the search results? There is no "npm ci" on the first page, and all the usages are actually about npm run ci:test or something along those lines, which is exactly why I suggested to be explicit about npm clean-install since I was confused how npm could do anything related to continuous integration as its base command.

Apparently, search results change every now and then, I just looked it up again - and about a half of results on the first page included `npm ci` literally.

> These are all legit CI pipelines where clean-install is used explicitly

That's true, case of using "npm" and "clean-install" separately in one file is a rare one.

Maybe that's not a good metric.
Can you come up with a metric that can resolve our argument? But something you don't check out before proposing ;)
Something like "npm ci vs npm clean-install quoted in google - amount of results" or "top js/ts repos on github including either of this terms - whichever option gets 10 repos first" :)

---

Let's use this metric: we have two people in the room now, and one finds `npm ci` confusing, but it seems that none of us find `npm clean-install` confusing. It is not clear why we need to have this argument if it does not cost us a bit to be more explicit using the full name of the command. Do you believe explicitness hurts anyone here?

I suggest we don't spend any more time on it. Make your best call, and let's go with it as it is not that big of a deal for me.

---

> Let's use this metric: we have two people in the room now, and one finds npm ci confusing, but it seems that none of us find npm clean-install confusing.

Hey, I asked for a metric you can't check out before proposing! :)

> Do you believe explicitness hurts anyone here?

Nope, I just enjoy having existential discussions from time to time.
Sorry for spending too much time on that, I just wanted to pick some metric! :)